tiled/adapters/table.py

```diff
@@ -57,6 +57,38 @@ def from_pandas(
             ddf, metadata=metadata, specs=specs, access_policy=access_policy
         )
 
+    @classmethod
+    def from_pydict(
```

Comment:
Little naming thing: the method name `from_dict` is way more common. (See `awkward`, `pandas`, `polars`.) Arrow's choice of `from_pydict` is a bit quirky and maybe reflects Arrow's perspective as a C++ library that it is interfacing with Python here.

Reply:
Changed and tested in incoming PR